### ISSUE TITLE
Issue with install and branches

### ISSUE THREAD

**Issue by chrisschaub** [handshake-org/hsd#713]
This command from the install
```
git clone --depth 1 --branch latest https://github.com/handshake-org/hsd.git
```
no longer works due to "latest" not being found? What is the best branch to track if we want to run an hsd node in our backend. Thanks.

**Comment by pinheadmz** [comment]
Hi Sorry we just merged our new release guidelines (#663) and haven't actually made the repo totally compliant with our own guidelines yet. This is current latest: https://github.com/handshake-org/hsd/releases/tag/v3.0.1

**Comment by nodech** [comment]
Okay, v3.0.1 has been tagged as `latest`. You can use either `v3.0.1` (current latest version) or `latest`.
